blog/2023-06-14-paralus-at-platformcon-2023/index.md

```diff
@@ -0,0 +1,114 @@
+
+## Entering Paralus, centralized RBAC and Authentication
+
+Paralus is an open-source project that focuses on RBAC and authentication in Kubernetes environments. It offers a single pane of glass for RBAC management and enhances security through centralized control. 
+
+<img src="/img/docs/paralus-platformcon-2023.png" alt="how paralus works" height="90%" width="90%"/>  
+
+
+## Zero-Trust K8s Access with Paralus
+
+Paralus grants authorized users seamless and secure access to all clusters with a native and familiar kubectl experience by acting as a proxy between the users and systems needing access and the K8s API server. It also addresses one of K8s' main pain points by eliminating the burden of managing K8s access controls cluster by cluster. Without Paralus, companies must manually manage access to each cluster using jump hosts or VPNs, and build custom tooling to audit and map all actions performed to a user's identity – all of which which is error-prone and increases the risk of breaches as the number of clusters grows.  
+
+**Thanks to Paralus, we can achieve the following outcomes**:
+
+**Secure Access Federation**: Paralus acts as the entry point for accessing all managed clusters, ensuring secure authentication and authorization across the board. It enforces consistent access controls and policies, reducing the risk of unauthorized access.
+
+**RBAC Simplification**: Paralus provides a centralized interface for managing RBAC across all clusters, enabling administrators to define and enforce role-based access at scale. This streamlines the process of granting permissions and ensures consistent authorization policies for developers.
+
+**Audit Logging and Compliance**: Paralus captures detailed audit logs of user activities, allowing organizations to meet compliance requirements and investigate any security incidents or policy violations efficiently.
+
+## Conclusion 
+
+Managing multiple Kubernetes clusters and federating access while maintaining robust RBAC practices is a complex task. However, Paralus offers a comprehensive solution that simplifies cluster management, enhances security, and streamlines RBAC operations.
+
+Paralus' centralized RBAC and authentication capabilities, organizations can effectively manage their clusters, achieve audit logging, and implement advanced RBAC practices, ultimately leading to improved efficiency, security, and compliance.
+
+### PlatformCon 2023 More Trends, Better Collaborations, Future of Platform Engineering is bright 
+
+PlatformCon 2023 was packed with internal platform use cases, blueprints and stories from a wide range of companies and platform practitioners. Speakers revealed the continued excited journey, roadmap to succeeding at platform engineering with better developer productivity and accelerated product development.
```

Comment:
Please rephrase this, it doesn't convey the message well.

Reply:
In this section I'm trying to highlight trends as it co-relates to our work.

Reply:
`Speakers revealed the continued excited journey, roadmap to succeeding at platform engineering with better developer productivity and accelerated product development.`

This part isn't clear. 

Maybe we can rephrase it to `Developers shared their learnings and tips to succeed at platform engineering by improving developer productivity and product development.`